src/model/quotes.js

```diff
@@ -66,32 +68,21 @@ class QuotesModel {
     this.requestId = config.requestId
   }
 
-  async executeRules (headers, quoteRequest) {
+  async executeRules (headers, quoteRequest, payer, payee) {
     if (rules.length === 0) {
       return []
     }
 
-    // Collect facts to supply to the rule engine
-    // Get quote participants from central ledger admin
-    const { switchEndpoint } = new Config()
-    const url = `${switchEndpoint}/participants`
-    const [payer, payee] = await Promise.all([
-      axios.request({ url: `${url}/${headers['fspiop-source']}` }),
-      axios.request({ url: `${url}/${headers['fspiop-destination']}` })
-    ])
-
-    this.writeLog(`Got rules engine facts payer ${payer} and payee ${payee}`)
-
     const facts = {
-      payer,
-      payee,
+      payer: payer.data,
```

Comment:
Why `payer.data` and `payee.data` here? It strikes me that `fetchParticipantInfo` should return the payer and payee, not the axios response objects.